examples/posenet/src/PoseNet.js

```diff
@@ -0,0 +1,372 @@
+        dimension_in = dimension_out;
+        this._outputs.push(this._operandIndex);
+        this._model.addOperand({type: type, dimensions: dimension_out});
+        this._operandIndex++;
+      }
+      if(this._tfmodel[i]["convType"] === "separableConv"){
+        let data = await getDimensionData("separableConv", this._version, i);
+        if(this._tfmodel[i].rate == 1){
+          dimension.push(resize(data[0][0]));
+          dimension.push(resize(data[0][1]));
+          weights.push(new Float32Array(transpose_weights(data[1][0], data[0][0])));
+          weights.push(new Float32Array(transpose_weights(data[1][1], data[0][1])));
+          dimension_bias.push(data[2][0]);
+          dimension_bias.push(data[2][1]);
+          bias.push(data[3][0]);
+          bias.push(data[3][1]);
+        }else{
+          let dilationData = dilationWeights(new Float32Array(transpose_weights(data[1][0], data[0][0])), 
+                                              resize(data[0][0]), this._tfmodel[i].rate);
+          dimension.push(dilationData[0]);
+          dimension.push(resize(data[0][1]));
+          weights.push(dilationData[1]);
+          weights.push(new Float32Array(transpose_weights(data[1][1], data[0][1])));
+          dimension_bias.push(data[2][0]);
+          dimension_bias.push(data[2][1]);
+          bias.push(data[3][0]);
+          bias.push(data[3][1]);
+        }
+
+        if(this._tfmodel[i].rate == 1){
+          dimension_out = this._calculateOutput(dimension_in, dimension[0], 
+                                                this._tfmodel[i]["stride"], 
+                                                "depthwise");
+        }
+        else{
+          dimension_out = this._calculateOutput(dimension_in, resize(data[0][0]), 
+                                                this._tfmodel[i]["stride"], 
+                                                "depthwise");
+        }
+        dimension_in = dimension_out;
+        this._outputs.push(this._operandIndex);
+        this._model.addOperand({type:type, dimensions: dimension_out});
+        this._operandIndex++;
+        dimension_out = this._calculateOutput(dimension_in, dimension[1], 1, "pointwise");
+        dimension_in = dimension_out;
+        this._outputs.push(this._operandIndex);
+        this._model.addOperand({type:type, dimensions: dimension_out});
+        this._operandIndex++;
+      }
+      for(let j in dimension){
+        let tensorId = this._operandIndex++;
+        let tensorType = {type: type, dimensions: dimension[j]};
+        this._shape.push(dimension[j]);
+        this._model.addOperand(tensorType);
+        this._model.setOperandValue(tensorId, weights[j]);
+        tensorId = this._operandIndex++;
+        let tensorType_bias = {type: type, dimensions: dimension_bias[j]};
+        this._model.addOperand(tensorType_bias);
+        this._model.setOperandValue(tensorId, bias[j]);
+      }      
+    }
+    //insert output layer
```

Comment:
For comments, please leave one space between `//` and following words, e.g. `// Insert output layers`. Apply to all comments.